ui/src/components/executions/Gantt.vue

```diff
@@ -367,7 +374,7 @@
                 }
 
                 > * {
-                    padding: calc(var(--spacer) / 2);
+                    padding: 1rem calc(var(--spacer) / 2);
```

Comment:
I do not think there isa need to keep the spacer here. we should use rem instead to be clearer

```suggestion
                    padding: 1rem .5rem;
```